godot-bevy/src/plugins/assets.rs

```diff
@@ -273,16 +272,7 @@ impl AssetLoader for GodotResourceAssetLoader {
             "res", "tres", // Resources
             "jpg", "jpeg", "png", // Images
             "wav", "mp3", "ogg", "aac", // Audio
+            "", // uid
```

Comment:
I'm not sure this is necessary. It doesn't seem to effect the ability for the uid loader to work.